test_unstructured_ingest/expected-structured-output/Sharepoint/None

```diff
@@ -1,25 +1,25 @@
 [
   {
-    "element_id": "32bc8af17151389d3e80f65036f8e65b",
+    "type": "Table",
```

Comment:
Also renamed to None. Probably not what we want.

Reply:
Good catch, this was a minor issue in the file name. Should be fixed now.